nectar/downloaders/threaded.py

```diff
@@ -160,6 +160,7 @@ def requests_kwargs_from_nectar_config(config):
                                          host=parsed_url.host, port=config.proxy_port)
             requests_kwargs['proxies'] = {'http': parsed_url.url, 'https': parsed_url.url}
 
+        requests_kwargs['stream'] = True # required for reading the download in chunks
```

Comment:
oh ok.  I can test that tomorrow.  Other than pulp-streamer, are there any other users of nectar?  (if this is a regression in this project, shouldn't it be fixed here??)
I'm not using Katello, but it looks as if the latest release (3.3) will be hitting this bug.  They ship nectar 1.5.3 but use pulp 2.10 [by the looks of things](https://fedorapeople.org/groups/katello/releases/yum/3.3/pulp/el7/x86_64/).

In fact, 1.5.3 is also in the *official* pulp repos for [2.10](https://repos.fedorapeople.org/repos/pulp/pulp/stable/2.10/7/x86_64/) and [2.11](https://repos.fedorapeople.org/repos/pulp/pulp/stable/2.11/7/x86_64/).

If done in the streamer, I guess the fix should be backported to those 2 releases too.